app/models/apartment.rb

```diff
@@ -0,0 +1,4 @@
+class Apartment < ActiveRecord::Base
+  validates :renter, :phone1, presence: true 
+  validates :phone1, format: { without: /[a-zA-Zа-яА-Я]/, message: 'Номер телефона не должен содержать буквы'}
```

Comment:
Line is too long. [110/80]<br>Space inside } missing.<br>Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.